include/OpenSpaceToolkit/Astrodynamics/Dynamics/Thruster/GuidanceLaw.hpp

```diff
@@ -0,0 +1,103 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Dynamics_Thruster_GuidanceLaw__
+#define __OpenSpaceToolkit_Astrodynamics_Dynamics_Thruster_GuidanceLaw__
+
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+#include <OpenSpaceToolkit/Core/Types/Shared.hpp>
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
+
+#include <OpenSpaceToolkit/Physics/Coordinate/Frame.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Instant.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace dynamics
+{
+namespace thruster
+{
+
+using ostk::core::types::String;
+using ostk::core::types::Shared;
+using ostk::core::types::Real;
+
+using ostk::math::obj::Vector3d;
+
+using ostk::physics::coord::Frame;
+
+using ostk::physics::time::Instant;
+
+/// @brief                      An interface for a Guidance Law that can computes an acceleration contribution. To be
+/// used in conjunction with a Thruster class to propagate a satellite to a target orbit.
+
+class GuidanceLaw
+{
+   public:
+    /// @brief                  Constructor
+    ///
+    /// @param                  [in] aName A name
+
+    GuidanceLaw(const String& aName);
+
+    /// @brief                  Destructor
+
+    virtual ~GuidanceLaw();
```

Comment:
This is just good practice to make this "base" class destructor virtual, correct?

Reply:
Yup precisely.